btree/btree.c

```diff
@@ -7855,21 +7946,36 @@ static void btree_ut_kv_oper_thread_handler(struct btree_ut_thread_info *ti)
 		ut_cb.c_datum = &data;
 
 		while (key_first <= key_last) {
+			/**
+			 * for variable key/value size, the size will increment
+			 * in multiple of 8 after each iteration. The size will
+			 * roundup on reaching MAX_KEY_SIZE. To make sure
+			 * there is atleast one key and size, arr_count is
+			 * incremented by 2.
+			 */
+			arr_count = (key_first % KV_ARR_SIZE) + 2 ;
+			ksize = ksize_random ?  arr_count * sizeof(key[0]):
+						ti->ti_key_size;
+			vsize = vsize_random ?  arr_count * sizeof(value[0]) :
+						ti->ti_value_size;
```

Comment:
put M0_ASSERT(ksize<=MAX_KEY_SIZE && vsize<=MAX_VAL_SIZE) as in kv_get we are also checking this condition ,

Reply:
ok.